addons/im_livechat/tests/test_im_livechat_support_page.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+from odoo.tests import HttpCase
+
+TEST_READY_JS_CODE = """
+if (document.readyState === "complete" && !window.testLivechatSetupHasRun) {
+    window.testLivechatSetupHasRun = true;
+    console.log("in setup");
+    odoo.__DEBUG__.didLogInfo.then(() => {
+        console.log("in then");
+        window.testLivechatIsReady = true;
+    });
+}
+window.testLivechatIsReady;
+"""
+
+CHECK_MODULES_JS_CODE = """
+const { missing, failed, unloaded } = odoo.__DEBUG__.jsModules;
+if ([missing, failed, unloaded].some(arr => arr.length)) {
+    console.error("Couldn't load all JS modules.")
+    console.error({ missing, failed, unloaded });
+} else {
+    console.log("test successful.");
+}
+"""
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestImLivechatSupportPage(HttpCase):
+    def test_load_modules(self):
+        """Checks that all javascript modules load correctly on the livechat support page"""
+        # We use a dummy name because this is not a real tour, the JS code above is the real test.
+        self.start_tour("/im_livechat/support/1", "dummy_name", code=CHECK_MODULES_JS_CODE, ready=TEST_READY_JS_CODE)
```

Comment:
We could use `browser_js` directly if the other params of `start_tour` are useless.